svm-transaction/src/tests.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg(test)]
+#![allow(clippy::arithmetic_side_effects)]
```

Comment:
why did this become necessary?

Reply:
was just lazy when copying in some message creation code. I replaced the offending arithmetic with saturating and checked methods instead: https://github.com/anza-xyz/agave/pull/5555/commits/cf4f96a308ff20981efa7ab61238aa6ff45b14d6